sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -232,6 +232,25 @@ class WholeStageCodegenSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-24901 check merge FastHashMap and RegularHashMap generate code max size") {
```

Comment:
This test looks weird, it's tricky to check the generated code size as it can change later.